oracle/types/market.go

```diff
@@ -53,14 +53,34 @@ func ProviderTickersFromMarketMap(
 type TickersToProviderTickers map[mmtypes.Ticker]DefaultProviderTicker
```

Comment:
Why use a `Ticker` as the index to a data-structure used in providers? The entire point here is to decouple providers / market-map types? If we need this structure, why can't the index be the offChainTicker?

Reply:
We still need a mapping of on-chain to off-chain rep so that our providers can have local configs that we can use for testing and such. I do think that we could probably simplify it to use just currencypair which omits much of the other info that Ticker includes.